README.md

```diff
@@ -270,6 +270,10 @@ Unset a value with explicit null:
 
     b3scalectl set frontend -j '{"default_presentation": {"url": "https://..."}}' frontend1
 
+### Configure a limit on overall attendees
+
+    b3scalectl set frontend -j '{"limit_attendees": {"limit": 50}}' frontend1
+
```

Comment:
Final note: Anni and I had a discussion about this and share the feeling that `limit_attendees.limit` "feels" wrong/inconsitent. A better naming might be `attendees_limit.limit` (declarative, not imperative, consistent with the rest of b3scale) or just `attendees_limit` (DRY, unless there are other settings in this context that you have in mind, in which case `attendees.limit`might be better).

I apologize because know this might inconvenience you with an existing setup, but will be cleaner down the line.

Reply:
(Sidenote: We had to release v1.1.1 because of important fixes, but I'd love to do a new release dedicated to this feature once this lands.

Reply:
Also, I really need to cut down on the README and point to the docs, which are now superior. Maybe (also) document this feature there?

Reply:
I am very sorry that it took some time. We've renamed it to AttendeesLimit/attendees_limit but kept the structure as we could imagine additional settings e.g. soft and hard limits. I've also added the new setting to the docs.

We'd be happy to get this in the next release.